.github/workflows/ci.yml

```diff
@@ -145,6 +145,28 @@ jobs:
           pip install tox-uv
           tox -e audit-${{ matrix.py-version.tox }}
 
+  singlepointprecision:
```

Comment:
as discussed lets remove CI and tox until al tests can be run in singlepoint precision

Reply:
Removed.